charts/neuvector/100.0.1+up2.2.2/README.md

```diff
@@ -0,0 +1,224 @@
+
+```console
+$ kubectl get clusterrole cluster-admin
+```
+
+If nothing returned, then add the `cluster-admin`:
+
+cluster-admin.yaml
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-admin
+rules:
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- nonResourceURLs:
+  - '*'
+  verbs:
+  - '*'
+```
+
+```console
+$ kubectl create -f cluster-admin.yaml
+```
+
+- If you have not created a service account for tiller, and give it admin abilities on the cluster:
+
+```console
+$ kubectl create serviceaccount --namespace kube-system tiller
+$ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+$ kubectl patch deployment tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}' -n kube-system
+```
+
+## CRD
+Because the CRD (Custom Resource Definition) policies can be deployed before NeuVector's core product, a new 'crd' helm chart is created. The crd template in the 'core' chart is kept for the backward compatibility. Please set 'crdwebhook.enabled' to false, if you use the new 'crd' chart.
+
+## Choosing container runtime
+The NeuVector platform supports docker, cri-o and containerd as the container runtime. For a k3s/rke2, or bottlerocket cluster, they have their own runtime socket path. You should enable their runtime options, k3s.enabled and bottlerocket.enabled, respectively.
+
+## Configuration
+
+The following table lists the configurable parameters of the NeuVector chart and their default values.
+
+Parameter | Description | Default | Notes
+--------- | ----------- | ------- | -----
+`openshift` | If deploying in OpenShift, set this to true | `false` |
+`registry` | NeuVector container registry | `registry.neuvector.com` |
+`tag` | image tag for controller enforcer manager | `latest` |
+`oem` | OEM release name | `nil` |
+`imagePullSecrets` | image pull secret | `nil` |
+`rbac` | NeuVector RBAC manifests are installed when rbac is enabled | `true` |
+`psp` | NeuVector Pod Security Policy when psp policy is enabled | `false` |
+`serviceAccount` | Service account name for NeuVector components | `default` |
+`controller.enabled` | If true, create controller | `true` |
+`controller.image.repository` | controller image repository | `neuvector/controller` |
+`controller.image.hash` | controller image hash in the format of sha256:xxxx. If present it overwrites the image tag value. | |
+`controller.replicas` | controller replicas | `3` |
+`controller.schedulerName` | kubernetes scheduler name | `nil` |
+`controller.affinity` | controller affinity rules  | ... | spread controllers to different nodes |
+`controller.tolerations` | List of node taints to tolerate | `nil` |
+`controller.resources` | Add resources requests and limits to controller deployment | `{}` | see examples in [values.yaml](https://github.com/neuvector/neuvector-helm/blob/5.0.0/charts/core/values.yaml)
```

Comment:
Should references in `README.md` be updated to point to the 2.2.2 tag? or is the 5.0.0 branch also used for v5.0.2 of NeuVector?
```suggestion
`controller.resources` | Add resources requests and limits to controller deployment | `{}` | see examples in [values.yaml](https://github.com/neuvector/neuvector-helm/tree/2.2.2/charts/core/values.yaml)
```

Reply:
I will make changes as you suggested and resubmit. Both have the same content, it is better to point to the correct chart version file.

Reply:
I used to do the below steps to resubmit, is it the correct way?
reset the commit 
recreate the chart (package, asset and chart)
reopen PR again

Reply:
Next time you can leave the PR open and just force push the new commits to the same upstream branch in your fork. This can be done by resetting the commit and recreating charts as you did, or you can update commits by creating a new local branch with the following steps.

   - Create a new local branch from dev-v2.6
   - recreate the charts
   - set the upstream to the forked branch used in this PR
   - force push (old commits will be overridden with those in the new branch)
   
Charts PRs are a little more involved since there are multiple commits and generated config files. If you are updating a PR with a single commit, you could do the following:
- `git add <changed_files>` 
- `git commit --amend` 
- `git push -f`

After your desired commits are updated, re-request reviews, and your reviewers will know you're ready.

Reply:
Thanks for your suggestion.  PR is closed as soon I reset the commit. I will try other method next time.